pkg/modprobe/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 AS mirror
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    busybox
```

Comment:
@rn just keeping busybox; musl installed already as a dep. Please review and merge if all looks good